docs/com.bitwarden.organizations.md

```diff
@@ -0,0 +1,79 @@
+[Table of contents](README.md#table-of-contents)
+
+# Bitwarden organization
+
+The `com.bitwarden.organization` doctype is used to group
+[ciphers](./com.bitwarden.ciphers.md) in a single unit and/or to share them with
+other [bitwarden contacts](./com.bitwarden.contacts.md).
+
+## Attributes
+
+- `name`: {string} - the name of the organization
+- `members`: {object} - the members of the organization (see below)
+- `defaultCollection`: {object} - the collection of the organization (see below)
+- `cozyMetadata`: {object} - the [cozyMetadata](README.md#document-metadata)
+
+## Members
```

Comment:
it seems that this is not correctly displayed on github

Reply:
Ok, took me a few minutes to understand what happened: 
- I have a windows azerty keyboard on macos
- So to type `#` I should type `alt gr`+`3`
- In macos `alt gr`+`space` inserts a no-break space

So I just didn't release `alt gr` in time before pressing the space bar 😅